src/rdkafka.c

```diff
@@ -4794,7 +4794,9 @@ static void rd_kafka_ListGroups_resp_cb(rd_kafka_t *rk,
 
                 state->wait_cnt++;
                 error = rd_kafka_DescribeGroupsRequest(
-                    rkb, 0, grps, i, RD_KAFKA_REPLYQ(state->q, 0),
+                    rkb, 0, grps, i,
+                    rd_false /* include_authorized_operations */,
```

Comment:
I think we should come up with a convention for these comments. Somewhere we try to explain what we are doing and somewhere we are just adding the field name. Sometimes its confusing what we are telling.

Reply:
I think, in this case, it's because the value doesn't make it obvious what the parameter is exactly, so for easy reading, we're adding the field name.
I think it's worth just always using the field name rather than describing what we're doing unless our behaviour is unusual in some way. I can make this change elsewhere also. What do you think?